app/templates/courses/teacher-class-view.jade

```diff
@@ -289,7 +289,7 @@ mixin studentLevelProgressDot(progress, level, levelNumber, student)
       - dotClass += ' practice'
 
 
-  if link && !state.get('readOnly')
+  if link
```

Comment:
Same as above - Would this allow SA to click on just the project dot, or all the course progress dots?

Reply:
The SA should be able to view all the things a teacher can view, so we want to open this up completely to the SA.